manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ShortestPathPlanningTest.scala

```diff
@@ -69,25 +79,46 @@ class ShortestPathPlanningTest extends DocumentingTest {
       }
     }
     section("Shortest path with additional predicate checks on the paths") {
-      p("""Predicates used in the `WHERE` clause that apply to the shortest path pattern are evaluated before deciding
-           |what the shortest matching path is. """)
-      query(
-        """MATCH (cs:Person {name:"Charlie Sheen"}),
-          |      (ms:Person {name:"Martin Sheen"}),
-          |      p = shortestPath( (cs)-[*]-(ms) )
-          |WHERE length(p) > 1
-          |RETURN p""", assertShortestPathLength) {
+      section("Consider using the exhaustive search as a fallback") {
         p(
-          """This query, in contrast with the one above, needs to check that the whole path follows the predicate
-            |before we know if it is valid or not, and so the query plan will also include the fall back to the slower
-            |exhaustive search algorithm""")
-        profileExecutionPlan()
+          """Predicates used in the `WHERE` clause that apply to the shortest path pattern are evaluated before deciding
+            |what the shortest matching path is. """)
+        query(
+          """MATCH (cs:Person {name:"Charlie Sheen"}),
```

Comment:
We are inconsistent on this page (as in other places) with using single and double quotes for string literals. While not part of the purpose of this PR, since the style guide now favors single quotes, if you have time to make the changes to align all the string literals it would be a nice bonus.